pymc/tests/test_shape_handling.py

```diff
@@ -323,7 +323,7 @@ def test_simultaneous_size_and_dims(self, with_dims_ellipsis):
             assert "ddata" in pmodel.dim_lengths
 
             # Size does not include support dims, so this test must use a dist with support dims.
-            kwargs = dict(name="y", size=2, mu=at.ones((3, 4)), cov=at.eye(4))
+            kwargs = dict(name="y", size=(2, 3), mu=at.ones((3, 4)), cov=at.eye(4))
```

Comment:
The `mu` creates an implied dimension which is now required to be included in `size`.